### PR TITLE
Allow Xcode version to by specified with ios_xc_test_args

### PR DESCRIPTION
**Pull request by LukeDurrant**

### CODE REVIEW

**Comment by nancy-gomez** [comment]
Hey @powerivq  do you think you can take a look and move this PR along? :) It addresses https://github.com/fastlane/fastlane-plugin-firebase_test_lab/issues/64 :D

File: lib/fastlane/plugin/firebase_test_lab/options.rb [hunk 1]
```diff
@@ -75,7 +75,13 @@ def self.available_options
                                        verify_block: proc do |value|
                                          v = File.expand_path(value.to_s)
                                          UI.user_error!("Key file not found at path '#{v}'") unless File.exist?(v)
-                                       end)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :ios_xc_test_args,
+                                       description: "extra ios_xc_test_args useful for xcode version ie { xcodeVersion: '10.1' }" \
```
Comment (powerivq) [inline lib/fastlane/plugin/firebase_test_lab/options.rb]: Suggested description:
```
Xcode version for running the tests.
```

This description is seen by the users and should be self explanatory.

File: lib/fastlane/plugin/firebase_test_lab/options.rb [hunk 2]
```diff
@@ -75,7 +75,13 @@ def self.available_options
                                        verify_block: proc do |value|
                                          v = File.expand_path(value.to_s)
                                          UI.user_error!("Key file not found at path '#{v}'") unless File.exist?(v)
-                                       end)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :ios_xc_test_args,
```
Comment (powerivq) [inline lib/fastlane/plugin/firebase_test_lab/options.rb]: Creating a single param for the xcode version seems enough? We don't want to expose the structure of the request payload if not necessary.

File: lib/fastlane/plugin/firebase_test_lab/module.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module Fastlane
   module FirebaseTestLab
-    VERSION = "1.0.5"
+    VERSION = "1.0.7"
```
Comment (powerivq) [inline lib/fastlane/plugin/firebase_test_lab/module.rb]: 1.0.6

File: lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb [hunk 4]
```diff
@@ -103,7 +104,7 @@ def self.wait_for_test_results(ftl_service, gcp_project, matrix_id, async)
 
               if async
                 UI.success("Job(s) have been submitted to Firebase Test Lab")
-                return
+                return firebase_console_link
```
Comment (powerivq) [inline lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb]: Why making this change? Don't see how it's been used.

File: lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb [hunk 5]
```diff
@@ -144,7 +145,7 @@ def self.wait_for_test_results(ftl_service, gcp_project, matrix_id, async)
               UI.test_failure!("Tests failed. " \
                 "Go to #{firebase_console_link} for more information about this run")
             end
-            return
+            return firebase_console_link
```
Comment (powerivq) [inline lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb]: Ditto

**Comment by powerivq** [comment]
@LukeDurrant Hi, thanks for your first contribution! I think this is going to be useful. I left some comments for the PR.

**Comment by nancy-gomez** [comment]
Hey @LukeDurrant there are some comments on this PR that need to be addressed before it can be merged :)

File: lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb (hunk 4 above)
Comment (LukeDurrant) [reply lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb]: We want the firebase console link. We post to slack after the job has been successful

File: lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb (hunk 5 above)
Comment (LukeDurrant) [reply lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb]: Same as above

File: lib/fastlane/plugin/firebase_test_lab/options.rb (hunk 2 above)
Comment (LukeDurrant) [reply lib/fastlane/plugin/firebase_test_lab/options.rb]: Theres other params that can be added in xc_test_args, seems like a waste of time having individual params for each one.

File: lib/fastlane/plugin/firebase_test_lab/options.rb (hunk 1 above)
Comment (LukeDurrant) [reply lib/fastlane/plugin/firebase_test_lab/options.rb]: its not just for Xcode version for running tests though is it?

**Pull request closed without merging**